auto_editor/Fix1003Rule.py

```diff
@@ -17,5 +18,105 @@ def change_type(self) -> ChangeTypeEnum:
     def run_rule(self, project: StructuredProjectSource,
                  warning_first_line: int, warning_last_line: int, file_path: str) -> StructuredProjectSource:
         # Todo: add rule here
-        print("Add rule here")
+        first_time = True
+        warning_code, prefix = "", ""
+        tmp_dict = project.paths_to_lines
+        all_lines = tmp_dict[file_path]
+
+        for i in range(warning_last_line+1, len(all_lines)):
+            print("warning code:",all_lines[i].code)
+            if ";" not in all_lines[i].code:
+                now_code = self.strip_determin(first_time, all_lines[i])
+                warning_code = warning_code + now_code
+                warning_code = warning_code.replace("\n", "")
+                first_time = False
+            else:
+                now_code = self.strip_determin( first_time, all_lines[i])
+                warning_code = warning_code + now_code + "\n"
+                prefix, new_code = self.remove_function_info(warning_code)
+                new_code = prefix + new_code
+                print("new_code:",new_code)
+
+                self.replace_useless_multiple_line(warning_last_line, i, all_lines)
+                new_lineItem = LineItem(new_code)
+                new_lineItem.original_line = warning_last_line+1
```

Comment:
again, no need to assign .original_line, as this lineItem is new and not part of the original version